server/plugin.ts

```diff
@@ -43,6 +43,7 @@ import { getAuthenticationHandler } from './auth/auth_handler_factory';
 import { setupMultitenantRoutes } from './multitenancy/routes';
 import { defineAuthTypeRoutes } from './routes/auth_type_routes';
 import { createMigrationOpenSearchClient } from '../../../src/core/server/saved_objects/migrations/core';
+import { ReadOnlyCapabilitiesSwitcher } from './auth/read_only_capabilities_switcher';
```

Comment:
Why do we have a different capitalization convention for this one? it seems like every other one imported above starts with lower case